### PR TITLE
feat: Sysvar address check example added

### PR DESCRIPTION
**Pull request by anoushk1234**

### CODE REVIEW

File: programs/10-sysvar-address-checking/secure/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+use anchor_lang::prelude::*;
+
+declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
+
+#[program]
+pub mod insecure {
+    use super::*;
+
+    pub fn check_sysvar_address(ctx: Context<CheckSysvarAddress>) -> Result<()> {
+        msg!("Rent Key -> {}", ctx.accounts.rent.key().to_string());
```
Comment (armaniferrante) [inline programs/10-sysvar-address-checking/secure/src/lib.rs:10]: This should be renamed as "recommended".

Can we add an additional program under the "secure" title that manually does the rent address check.

Comment (anoushk1234) [reply programs/10-sysvar-address-checking/secure/src/lib.rs:10]: yea sure

**Comment by anoushk1234** [comment]
Added the manual check `require_eq!(ctx.accounts.rent.key(), sysvar::rent::ID);`

**Comment by armaniferrante** [comment]
Thank you!

**Comment by anoushk1234** [comment]
> Thank you!

It's always a pleasure working with the best in the solana ecosystem :)

**Pull request merged**